src/VisualStudio/Setup/source.extension.vsixmanifest

```diff
@@ -4,39 +4,14 @@
     <Identity Id="0b5e8ddb-f12d-4131-a71d-77acc26a798f" Version="|%CurrentProject%;GetBuildVersion|" Language="en-US" Publisher="Microsoft" />
     <DisplayName>Roslyn Language Services</DisplayName>
     <Description>C# and VB.NET language services for Visual Studio.</Description>
-    <ShortcutPath>..\CommonExtensions\Microsoft\ManagedLanguages\VBCSharp\LanguageServices</ShortcutPath>
+    <PackageId>Microsoft.CodeAnalysis.VisualStudio.Setup</PackageId>
   </Metadata>
   <Installation Experimental="true">
     <InstallationTarget Version="[14.0,]" Id="Microsoft.VisualStudio.Pro" />
     <InstallationTarget Version="[14.0,]" Id="Microsoft.VisualStudio.VSWinDesktopExpress" />
     <InstallationTarget Version="[14.0,]" Id="Microsoft.VisualStudio.VWDExpress" />
     <InstallationTarget Version="[14.0,]" Id="Microsoft.VisualStudio.VSWinExpress" />
   </Installation>
-  <Installer>
-    <Actions>
-      <Action Type="Ngen" Path="Microsoft.CodeAnalysis.CSharp.dll" />
-      <Action Type="Ngen" Path="Microsoft.CodeAnalysis.CSharp.EditorFeatures.dll" />
-      <Action Type="Ngen" Path="Microsoft.CodeAnalysis.CSharp.Features.dll" />
-      <Action Type="Ngen" Path="Microsoft.CodeAnalysis.CSharp.Workspaces.dll" />
-      <Action Type="Ngen" Path="Microsoft.CodeAnalysis.dll" />
-      <Action Type="Ngen" Path="Microsoft.CodeAnalysis.EditorFeatures.dll" />
-      <Action Type="Ngen" Path="Microsoft.CodeAnalysis.EditorFeatures.Text.dll" />
-      <Action Type="Ngen" Path="Microsoft.CodeAnalysis.Elfie.dll" />
-      <Action Type="Ngen" Path="Microsoft.CodeAnalysis.Features.dll" />
-      <Action Type="Ngen" Path="Microsoft.CodeAnalysis.VisualBasic.dll" />
-      <Action Type="Ngen" Path="Microsoft.CodeAnalysis.VisualBasic.EditorFeatures.dll" />
-      <Action Type="Ngen" Path="Microsoft.CodeAnalysis.VisualBasic.Features.dll" />
-      <Action Type="Ngen" Path="Microsoft.CodeAnalysis.VisualBasic.Workspaces.dll" />
-      <Action Type="Ngen" Path="Microsoft.CodeAnalysis.Workspaces.Desktop.dll" />
-      <Action Type="Ngen" Path="Microsoft.CodeAnalysis.Workspaces.dll" />
-      <Action Type="Ngen" Path="Microsoft.DiaSymReader.dll" />
```

Comment:
can't be NGEN'd its not a project reference, its a transitive dependency

---
In reply to: [91195497](https://github.com/dotnet/roslyn/pull/15710#discussion_r91195497) [](ancestors = 91195497)